formlibrary/serializers.py

```diff
@@ -45,4 +49,32 @@ class HouseholdListDataSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Household
-        fields = ['id', 'name', 'program', 'created_by', 'create_date']
+        fields = ['id', 'name', 'duration', 'program', 'created_by', 'create_date']
+
+
+class TrainingListDataSerializer(serializers.ModelSerializer):
+    program = ProgramSerializer()
```

Comment:
Why can't we move this to `TrainingSerializer` and let's keep only one serializer?
Same goes for DistributionListDataSerializer

Reply:
@nasirhjafri there is an issue when creating service for either `training` and `distribution` which is ```{
    "created_by": {
        "non_field_errors": "[ErrorDetail(string = Invalid data. Expected a dictionary, but got int.', code = 'invalid')]"
    },
    "program": {
        "non_field_errors": "[ErrorDetail(string = 'Invalid data. Expected a dictionary, but got int.', code = 'invalid')]"
    }
}``` caused when creating a service